tests/unit/test_event_handler.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from dbt.events.base_types import EventLevel
+from dbt.events.event_handler import DbtEventLoggingHandler, set_package_logging
+from dbt.events.eventmgr import TestEventManager
+
+
+def test_event_logging_handler_emits_records_correctly():
+    event_manager = TestEventManager()
+    handler = DbtEventLoggingHandler(event_manager=event_manager)
```

Comment:
It's in the test results too, but to save you the legwork, this is missing the `level` arg.